extensions/src/Smithy.Identity.Abstractions/Auth/IAuthSchemeParameters.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Smithy.Identity.Abstractions
+{
+    /// <summary>
+    /// <para>
+    /// The input to the authentication scheme resolver. A code-generated interface for passing 
```

Comment:
nit: We will code-generate it but since this is a Smithy interface that could someday be used by other users they could write it by hand. I suggest removing "code-generated" from the doc.